setup.py

```diff
@@ -35,7 +35,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk>=1.47.0",
-        "pennylane>=0.30.0",
+        "pennylane>=0.34.0",

```

Comment:
Just one more thing we would want to make sure that this should be breaking notebooks if we are using the older PL version there. ?

Reply:
We are using the latest PL version there for a while.

Reply:
https://github.com/amazon-braket/amazon-braket-examples/commit/f7f4ab40fc544bfd907cc6f1263c54222d3fffe9